src/main/java/com/cinema/shop/dao/CinemaHallDao.java

```diff
@@ -7,4 +7,6 @@
     CinemaHall add(CinemaHall cinemaHall);
 
     List<CinemaHall> getAll();
+
+    CinemaHall findById(Long id);
```

Comment:
what is the difference between  findById() and getById()?? let's use one type of naming